src/UnionModel.php

```diff
@@ -274,9 +274,6 @@ public function action($mode, $args = [])
             if ($this->getElement($field)->never_persist) {
                 continue;
             }
-            if ($this->getElement($field) instanceof \atk4\data\Expression_SQL) {
-                continue;
-            }
```

Comment:
yes. basically cleanup, because \atk4\data\Expression_SQL doesn't exist anyway.
It should be \atk4\data\Field_SQL_Expression, but ... we don't need it here.